### PR TITLE
Feature modal transition name

### PR DESCRIPTION
**Pull request by thomasJang**
…hod' argument property.

First of all, thank you for your contribution! :-)

Please makes sure that these checkboxes are checked before submitting your pull request, thank you!

* [X] Make sure that you propose pull request to right branch: bugfix for `master`, feature for branch `feature`.
* [X] Make sure that you follow antd's [code convention](https://github.com/ant-design/ant-design/wiki/Code-convention-for-antd).
* [X] Run `npm run lint` and fix those errors before submitting in order to keep consistent code style.
* [X] Rebase before creating a pull request to keep commit history clear.
* [X] Add some descriptions and refer relative issues for you pull request.

Extra checklist:

**if** *isBugFix* **:**

  * [ ] Make sure that you add at least one unit test for the bug which you had fixed.

**elif** *isNewFeature* **:**

  * [X] Update API docs for the component.
  * [ ] Update/Add demo to demonstrate new feature.
  * [ ] Update TypeScript definition for the component.
  * [ ] Add unit tests for the feature.

#12623

I respect the great open source "AntD" developers.
but I have a desire to be able to change the transitionName when using 'Modal.method'.

Thank you for your effort.

### CODE REVIEW

File: components/modal/index.en-US.md [hunk 1]
```diff
@@ -81,9 +81,12 @@ The properties of the object are follows:
 | zIndex | The `z-index` of the Modal | Number | 1000 |
 | onCancel | Specify a function that will be called when the user clicks the Cancel button. The parameter of this function is a function whose execution should include closing the dialog. You can also just return a promise and when the promise is resolved, the modal dialog will also be closed | function | - |
 | onOk | Specify a function that will be called when the user clicks the OK button. The parameter of this function is a function whose execution should include closing the dialog. You can also just return a promise and when the promise is resolved, the modal dialog will also be closed | function | - |
+| transitionName | className of modalTransition | string | `zoom` |
+| maskTransitionName | className of maskTransition | string | `fade` |
```
[the remark this answers is not in the document]
Comment (thomasJang) [reply components/modal/index.en-US.md]: I remove that code and push again

File: components/modal/index.en-US.md [hunk 2]
```diff
@@ -84,6 +84,7 @@ The properties of the object are follows:
 
 All the `Modal.method`s will return a reference, and then we can update and close the modal dialog by the reference.
 
+
```
Comment (afc163) [inline components/modal/index.en-US.md]: Don't change this.

Comment (picodoth) [reply components/modal/index.en-US.md]: ping @thomasJang

**Comment by afc163** [comment]
closing for no response

**Comment by vitalijalbu** [comment]
Hi guys, how to change transition of the modal? I don't like the zoom effect... want the slide-up
Using 5.5 version. Thanks

**Pull request closed without merging**